src/release/breaking-changes/key-event-migration.md

```diff
@@ -87,8 +89,9 @@ their functionality is no longer offered.
 
 ## Migration guide
 
-APIs provided by the Flutter framework are already migrated. Migration is needed
-only if any of the classes or methods listed in the previous section are used.
+APIs provided by the Flutter framework are already migrated.
+Migration is needed only if any of the classes or methods listed in
+the previous section are used.
```

Comment:
**chore:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
Flutter framework API have been migrated.
If your code includes any of the classes or methods listed in
the previous section, migrate to these new APIs.
```